Source/WebCore/rendering/RenderObject.h

```diff
@@ -1178,9 +1178,9 @@ class RenderObject : public CachedImageClient, public CanMakeCheckedPtr {
     private:
         uint32_t m_flags : 21 { 0 };
         uint32_t m_positionedState : 2 { IsStaticallyPositioned }; // PositionedState
-        uint32_t m_selectionState : 3 { HighlightState::None }; // HighlightState
-        uint32_t m_fragmentedFlowState : 1 { NotInsideFragmentedFlow }; // FragmentedFlowState
-        uint32_t m_boxDecorationState : 2 { NoBoxDecorations }; // BoxDecorationState
+        uint32_t m_selectionState : 3 { static_cast<uint32_t>(HighlightState::None) }; // HighlightState
```

Comment:
I like to use enumToUnderlyingType() instead of static_cast<>() for such purposes because then I don't need to worry about what the underlying type is.

Reply:
Ah, good point. Will do that.